config/build.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+set -euo pipefail
+
+GOCILINT_VERSION="1.31.0"
+GOCILINT_SHA256SUM="9a5d47b51442d68b718af4c7350f4406cdc087e2236a5b9ae52f37aebede6cb3"
+GOCILINT_LOCATION=https://github.com/golangci/golangci-lint/releases/download/v${GOCILINT_VERSION}/golangci-lint-${GOCILINT_VERSION}-linux-amd64.tar.gz
+
+OPERATOR_SDK_VERSION="0.16.0"
```

Comment:
This will make the image unusable for `test` from consumers using a different version once #41 is in place. (For building it's irrelevant, so fine.)